src/routes/cameraintro.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+import metrics from '../themes/metrics.js'
+
+import Navbar from '../components/Navbar.js'
+import FullWidthContainer from '../components/FullWidthContainer.js'
+import LargeText from '../components/LargeText.js'
+
+class CameraIntro extends React.Component {
+  render() {
+    return (
+      <View style={styles.pageContainer}>
```

Comment:
We could make page container a component if we are going to use this pattern in several places. Keep thinking how we can make things reusable.